index.php

```diff
@@ -95,7 +95,7 @@ private function build_index() {
 
 			$uri = trim( sprintf( 
 					'%s/%s',
-					trim( $uri_info['dirname'], '/' ),
+					trim( $uri_info['dirname'], '/\\' ),
```

Comment:
OK, great.

Could we please keep the `.md` parsing out of this pull request? `PHP_EOL` will use the `EOL` character of the server and not the document so it should be investigated further.